src/API/OnboardingTasks.php

```diff
@@ -610,4 +641,230 @@ public function get_status() {
+								'woocommerce-admin'
+							),
+						'content'       => sprintf(
+							// @todo This content should be updated based on products.
+							/* translators: %1$s: list of product names comma separated, %2%s the last product name */
+							__(
+								'Good choice! You chose to add %1$s and %2$s to your store.',
+								'woocommerce-admin'
+							),
+							$purchase_products[0],
+							$purchase_products[0]
+						),
+						'actionLabel'   => __( 'Purchase & install now', 'woocommerce-admin' ),
+						'actionUrl'     => '/setup-wizard',
+						'isComplete'    => count( array_diff( $purchase_products, $installed_plugins ) ) < 1,
+						'isVisible'     => count( $purchase_products ) > 0,
+						'time'          => __( '2 minutes', 'woocommerce-admin' ),
+						'isDismissable' => true,
+					),
+					array(
+						'id'         => 'products',
+						'title'      => __( 'Add my products', 'woocommerce-admin' ),
+						'content'    => __(
+							'Start by adding the first product to your store. You can add your products manually, via CSV, or import them from another service.',
+							'woocommerce-admin'
+						),
+						'isComplete' => 0 !== count( $products ),
+						'isVisible'  => true,
+						'time'       => __( '1 minute per product', 'woocommerce-admin' ),
+					),
+					array(
+						'id'          => 'woocommerce-payments',
+						'title'       => __( 'Get paid with WooCommerce Payments', 'woocommerce-admin' ),
+						'content'     => __(
+							"You're only one step away from getting paid. Verify your business details to start managing transactions with WooCommerce Payments.",
+							'woocommerce-admin'
+						),
+						'actionLabel' => __( 'Finish setup', 'woocommerce-admin' ),
+						'expanded'    => true,
+						'isComplete'  => $wc_pay_is_connected,
+						'isVisible'   => in_array( 'woocommerce-payments', $business_extensions, true ) &&
+							in_array( 'woocommerce-payments', $installed_plugins, true ) &&
+							in_array( WC()->countries->get_base_country(), OnboardingTasksFeature::get_woocommerce_payments_supported_countries(), true ),
+						'time'        => __( '2 minutes', 'woocommerce-admin' ),
+					),
+					array(
+						'id'         => 'payments',
+						'title'      => __( 'Set up payments', 'woocommerce-admin' ),
+						'content'    => __(
+							'Choose payment providers and enable payment methods at checkout.',
+							'woocommerce-admin'
+						),
+						'isComplete' => ! empty( $enabled_gateways ),
+						'isVisible'  => Features::is_enabled( 'payment-gateway-suggestions' ) &&
+							(
+								! in_array( 'woocommerce-payments', $business_extensions, true ) ||
+								! in_array( 'woocommerce-payments', $installed_plugins, true ) ||
+								! in_array( WC()->countries->get_base_country(), OnboardingTasksFeature::get_woocommerce_payments_supported_countries(), true )
+							),
+						'time'       => __( '2 minutes', 'woocommerce-admin' ),
+					),
+					array(
+						'id'          => 'tax',
```

Comment:
Since this list is extendable, I wonder if using more unique IDs would be worth-while to avoid collisions down the road?

Reply:
I'm not opposed to this, but since these IDs already existed, we would be breaking any backwards compatibility with filters being used.  But if we think they're not being used at all we can forego cc @octaedro